features/polyExplorer/src/components/clusterStories/mauLineChart.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from "react";
+import { ChipGroup, LineLegend, PolyChart } from "@polypoly-eu/poly-look";
+import i18n from "../../i18n";
+
+import "./mauLineChart.css";
+
+const facebookColor = "#3749A9";
+const otherColor = "#3BA6FF";
+
+const otherColorSet = { color: otherColor, gradient: "other-gradient" };
+const facebookColorSet = {
+    color: facebookColor,
+    gradient: "facebook-gradient",
+};
+
+const gradients = [
+    {
+        id: "facebook-gradient",
+        type: "linearGradient",
+        stops: [
+            { offset: "0%", color: facebookColor, opacity: 0.1 },
+            { offset: "100%", color: facebookColor, opacity: 0.5 },
+        ],
+    },
+    {
+        id: "other-gradient",
+        type: "linearGradient",
+        stops: [
+            { offset: "0%", color: otherColor, opacity: 0.1 },
+            { offset: "100%", color: otherColor, opacity: 0.5 },
+        ],
+    },
+];
+
+const coloring = {
+    Signal: otherColorSet,
+    Instagram: facebookColorSet,
+    WhatsApp: facebookColorSet,
+    Threema: otherColorSet,
+    Snapchat: otherColorSet,
+    "Facebook Messenger": facebookColorSet,
+    Telegram: otherColorSet,
+    TikTok: otherColorSet,
+    iMessage: otherColorSet,
+};
+
+const MauLineChart = ({ messengers, i18nHeader }) => {
```

Comment:
Naming nit: This doesn't seem to be particularly reusable, it's hard coding specific messengers, the ones from the messenger story. OTOH, I guess it doesn't really matter that much what _kind_ of chart this is, for anyone embedding this code. So should it be `MessengerMauChart` or something more in that vein?